transforms/svc_handler.sysl

```diff
@@ -321,16 +331,102 @@ CodeGenTransform:
             )
           )
         )
-
-        let hasPayload = if ep.value.ret.payload != "" then true else false
-
-        let returnValues = if hasPayload then "httpStatus, headerMap, " + ep.value.ret.payload else "httpStatus, headerMap"
-        let sendRespCallArgs = if hasPayload then "httpStatus, " + ep.value.ret.payload else "httpStatus"
-
-
+        let httpStatus = ep -> <Statement> (:
+          Statement = ep -> <Statement> (:
+            VarDecl = varDecl("httpStatus", "int")
+          )
+        )
+        let resultTypes = if ep.value.ret.payload == null then splitReturnTypes(ep.value.ret) else splitPayload(Split(ep.value.ret.payload, ', '))
+        let tempReturnValues = if resultTypes count > 0 then "headerMap, " + Join(resultTypes flatten(.typeName),", ") else "headerMap"
+        let returnValues = if ep.value.ret.payload == null then tempReturnValues else "Status, " + tempReturnValues
+        
+        let sendRespCallArgs = "httpStatus"
```

Comment:
Please use a better variable name and use it for all "httpStatus" args.

https://github.com/anz-bank/syslgen-examples/pull/7/files#diff-cde3e06ea4b813a0bc291f88402798f4R391
https://github.com/anz-bank/syslgen-examples/pull/7/files#diff-cde3e06ea4b813a0bc291f88402798f4R405